### ISSUE TITLE
Cannot find module 'js-import-sort/node_modules/jscodeshift/dist/package.json'

### ISSUE THREAD

**Issue by rosswarren** [Amwam/js-import-sort#20]
It looks like the library app-root-path is looking in js-import-sort for the package.json rather than the project I am trying to run it on.

I get the following error:

``` sh
Error: Cannot find module '/Users/ross/.nvm/versions/node/v6.4.0/lib/node_modules/js-import-sort/node_modules/jscodeshift/dist/package.json'
    at Function.Module._resolveFilename (module.js:455:15)
    at Function.Module._load (module.js:403:25)
    at Module.require (module.js:483:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/Users/ross/.nvm/versions/node/v6.4.0/lib/node_modules/js-import-sort/lib/thirdPartyModules.js:4:21)
    at Module._compile (module.js:556:32)
    at Object.Module._extensions..js (module.js:565:10)
    at Module.load (module.js:473:32)
    at tryModuleLoad (module.js:432:12)
    at Function.Module._load (module.js:424:3)
module.js:457
    throw err;
    ^
```

Changing to the library `find-root` fixes this for me and it works correctly.

``` js
const findRoot = require('find-root');
const packageJson = require(findRoot(process.cwd()) + '/package.json');
```

Any thoughts on this?

**Comment by Amwam** [comment]
Thats interesting, I'm not experiencing that problem. Could you detail some more about how you're running it?
Is it globally installed or locally?
Are you running it from inside you project's directory?

I've not seen 'find-root' before, and wouldn't mind switching to it, I'd just like to make sure I can replicate (and ideally write a test) for this case before doing so

**Comment by rosswarren** [comment]
I had installed it globally (in an nvm installation of node) and I was running it in a subfolder in my project directory. Is there a different way you would recommend running it?

**Comment by Amwam** [comment]
Ah, it currently expects to be run in the same level as the `package.json` location, although it should be more flexible. I'll try and replicate and update tomorrow. If you already have the change to `find-root` please feel free to submit a PR.
Thanks

**Comment by Amwam** [comment]
Fixed in 1.0.3. Thanks for you help!
